packages/components/src/form-token-field/test/index.tsx

```diff
@@ -486,6 +486,7 @@ describe( 'FormTokenField', () => {
 
 			// This is testing implementation details, but I'm not sure there's
 			// a better way.
+			// eslint-disable-next-line testing-library/no-node-access
```

Comment:
There's currently no accessible way to access the parent element or its parent element.